### PR TITLE
Enhancement/revamp trading sales page crw 330

### PR DESCRIPTION
**Pull request by AbdallaEmadEldinAbdalla**
[CRW-330](https://techxarya.atlassian.net/browse/CRW-330)

[CRW-330]: https://techxarya.atlassian.net/browse/CRW-330?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by alex-greenbull** [comment]
Screenshot_arya-crypto-web-git-enhancement-revamp-tradi-d9ea87-arya-crypto vercel appfrtrade_001424

This should be a carousell

**Comment by AbdallaEmadEldinAbdalla** [comment]
> Screenshot_arya-crypto-web-git-enhancement-revamp-tradi-d9ea87-arya-crypto vercel appfrtrade_001424
> 
> This should be a carousell

Me and AL will work on it

File: src/components/shared/Testimonials/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}, {
+    name: "Jeremie",
+    contnet: "Thank you ARYA, 100% of this software is great. Fortunately my wife is there to stop me at the right moment and bring me back to reality!",
+}, {
+    name: "Yann",
+    contnet: "Thanks to ARYA I had a great performance this month! My goal of the month is already reached.",
+}, {
+    name: "Matt",
+    contnet: "Great results, while I’m sleeping on the Dow Jones in M15 for a buy order I made last night! Epic.",
+}];
```
Comment (almujtabamohammed) [inline src/components/shared/Testimonials/index.tsx:39]: - Add translation
- Move to constants file

File: src/components/shared/Testimonials/index.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                }}
+                navigation={true}
+                modules={[Autoplay, Pagination, Navigation]}
+                className={styles.swiper}
+            >
+                {
+                    Arr.map((slide) => {
+                        return (
+                            <SwiperSlide key={slide.name}>
+                                <Col className="items-center bg-offWhite-1 rounded-2xl py-12 px-10 border-4 border-blue-4 h-[250px]">
```
Comment (almujtabamohammed) [inline src/components/shared/Testimonials/index.tsx:64]: Remove height, because it doesn't look good on small screens

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]

**Pull request merged**